### PR TITLE
Initial cut for django briefcase target

### PR DESCRIPTION
**Pull request by glasnt**
To be used with https://github.com/pybee/Python-Django-template. The how-to is in the 3.5 branch of the template

It's probably not 100%, there's still some Django-specifics that could be nicer, but it's a start. 

Feel free to djangoify it, @freakboy3742

### CODE REVIEW

**Comment by freakboy3742** [comment]
I've had a bit of a swing at this - it's cleaned up a few things. There's still a problem generating a `package.json` file that has content that doesn't raise warnings, but it's getting there.

File: briefcase/django.py [hunk 1]
```diff
@@ -43,13 +43,7 @@ def install_support_package(self):
     def install_extras(self):
         # Install additional elements required for Django
         print(" * Installing extras...")
-        print(" * Installing NPM requirements...")
-        package_url = "https://raw.githubusercontent.com/pybee/toga/master/src/django/package.json"
```
Comment (glasnt) [inline briefcase/django.py]: I wasn't sure if we should be using the toga-django install requirements, or a package.json like the source I was using from `togatest`. But it looks like you've added the package.json into the template, which works 👍

File: briefcase/app.py [hunk 2]
```diff
@@ -120,28 +134,41 @@ def find_support_pkg(self):
...
+        return os.path.join(os.getcwd(), self.resource_dir, 'app_packages')
+
     def generate_app_template(self):
         print(" * Writing application template...")
 
         if self.template is None:
             self.template = 'https://github.com/pybee/Python-%s-template.git' % self.platform
         print("Project template: %s" % self.template)
 
+        # import pdb; pdb.set_trace()
```
Comment (glasnt) [inline briefcase/app.py]: stray debugging

**Comment by glasnt** [comment]
🛰

**Pull request merged**